source/includes/fact-verifier-limitations.rst

```diff
@@ -0,0 +1,41 @@
+
+Compatibility
+~~~~~~~~~~~~~
+
+The embedded verifier is not available in mongosync 1.8 and
+earlier.
+
+For alternative verification methods, see :ref:`c2c-verification`.
+
+Limitations
+~~~~~~~~~~~
+
+The embedded verifier has the following limitations:
+
+- ``mongosync`` stores the verifier state in memory, which can
+  result in a significant memory overhead. To run verification
+  checks, ``mongosync`` requires approximately 0.5 GB of memory
+  for every 1 million documents.
+
+- Verification cannot be resumed. If a user stops or pauses sync
+  and then starts ``mongosync`` again for any reason, the
+  verification process restarts from the beginning. This can
+  cause verification to fall substantially behind the migration.
+
+- The verifier isn't compatible with :ref:`reverse sync
```

Comment:
The verifier is compatible with reverse enabled syncs in the forward direction. However, it will not run for the /reverse sync or any subsequent reversals.